install_config/storage_examples/index.adoc

```diff
@@ -10,3 +10,4 @@ The following sections provide detailed conprehensive instructions on setting up
 
 - link:../../install_config/storage_examples/shared_storage.html[Sharing an NFS PV Across Two Pods]
 - link:../../install_config/storage_examples/ceph_example.html[Ceph-RBD Block Storage Volume]
+- link:../../install_config/gluster_examples/gluster_example.html[Shared Storage Using a GlusterFS Volume]
```

Comment:
@jeffvance Dangit, just saw this right after I hit merge. Still shows "gluster_examples" in the path.